### PR TITLE
chore: bump to 2.5.12

### PR DESCRIPTION
**Pull request by ravijayaramappa**
## Details
Release commit for 2.5.12

## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

## GUS work item
W-10459160

### CODE REVIEW

**Review by jmsjtu — APPROVED** [review]

**Review by nolanlawson — APPROVED** [review]

File: lerna.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "exact": true,
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "2.5.11"
+  "version": "2.5.11-236.1"
```
Comment (nolanlawson) [inline lerna.json:5]: Historically we haven't used this format for OSS releases. Would it make more sense to release `2.5.12` instead?

Comment (ravijayaramappa) [reply lerna.json:5]: I looked into the history of the versions and found a few with this pattern: 2.2.10-234.0, 1.11.0-232.1, 1.10.1-232.1

I think the reasoning was that these are special branches specific to Salesforce releases. There could be cases when the next patch version could already be used by a release from master. (In the case of this PR, that is not an issue)

Comment (ravijayaramappa) [reply lerna.json:5]: Will fix that. I went by previous precedent. Spoke to @ekashida and he provided some additional context.

**Pull request merged**